psw/urts/loader.cpp

```diff
@@ -919,12 +919,19 @@ int CLoader::load_enclave_ex(SGXLaunchToken *lc, bool debug, const metadata_t *m
         case SGX_ERROR_ENCLAVE_LOST:     //caused by loading enclave while power transition occurs
             break;
 
-            //If memroy map conflict occurs, we only retry 3 times.
+            //If memroy map conflict or map fail occurs, we only retry 3 times.
         case SGX_ERROR_MEMORY_MAP_CONFLICT:
-            if(0 == map_conflict_count)
```

Comment:
I don't think lower level ever returns this error again. Could you remove this and update all related references to SGX_ERROR_MEMORY_FAILURE? For example in ae/aesm_service/source/oal/linux/internal_log.cpp, and all sample code.

Also mark it deprecated in sgx_error.h:
- SGX_ERROR_MEMORY_MAP_CONFLICT= SGX_MK_ERROR(0x2007),      /* Page mapping failed in driver */
+ SGX_ERROR_MEMORY_MAP_CONFLICT= SGX_MK_ERROR(0x2007),      /* Page mapping failed in driver. deprecated, replaced with SGX_ERROR_MEMORY_MAP_FAILURE */

Reply:
ok, I fix it.